src/controllers/docs.controller.ts

```diff
@@ -0,0 +1,29 @@
+import { NextFunction, Request, Response, Router } from 'express';
```

Comment:
We can remove the unused types:
```suggestion
import { Router } from 'express';
```